WcaOnRails/app/controllers/api/v0/roles_controller.rb

```diff
@@ -100,6 +116,12 @@ def index_for_group_type
     # Filter the list based on the permissions of the logged in user.
     roles = filter_roles_for_logged_in_user(roles)
 
+    # Filter the list based on the other parameters.
+    roles = filter_roles_for_parameters(
+      roles,
+      params.key?(:status) ? params.require(:status) : nil,
```

Comment:
Your line implies that the `status` param is not necessarily required, so it doesn't make sense to use `params.require`. A simple `params[:status]` should do